### PR TITLE
[REVIEW] Add codeowners file [skip ci]

### PR DESCRIPTION
**Pull request by dillon-cullinan**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#python code owners
+ucp/            @rapidsai/ucxpy-python-codeowners
+tests/          @rapidsai/ucxpy-dask-codeowners
```
Comment (jakirkham) [inline .github/CODEOWNERS]: Sorry if I'm missing something. Should this be `@rapidsai/ucxpy-python-codeowners` too?

**Comment by quasiben** [comment]
@dillon-cullinan  Do docs need owners as well ?  Also are the following listed as code owners:

jakirkham 
madsbk 
quasiben 
pentschev 
jacobtomlinson 
rjzamora 

 ?

**Comment by dillon-cullinan** [comment]
> @dillon-cullinan Do docs need owners as well ? Also are the following listed as code owners:
> 
> jakirkham
> madsbk
> quasiben
> pentschev
> jacobtomlinson
> rjzamora
> 
> ?

No to docs, yes to the list.

**Comment by quasiben** [comment]
benchmarks should probably be include, no ?

**Comment by jakirkham** [comment]
Thanks @dillon-cullinan! 😀

**Pull request merged**